### PR TITLE
Take screenshots of display on async start

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready| [Ticket](https://pi-top.atlassian.net/browse/FUR-2495) |

#### Main changes
`async_start` now returns a 'screenshot monitor' which continuously takes screenshots of the created virtual display and makes them available to user.

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: pt_web_vnc/screenshot_monitor.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import atexit
+from time import sleep
+from PIL import ImageGrab
```
Comment (angusjfw) [inline pt_web_vnc/screenshot_monitor.py:3]: PIL isn't added to the dependencies. this is also why the tests are failing

**Review by angusjfw — CHANGES_REQUESTED** [review]
looks great, just minor comments to resolve. it would also be good to add a test for this behaviour
